### ISSUE TITLE
MarkovChain solver should zero out dominated cycles

### ISSUE THREAD

**Issue by DrPaulBrewer** [DrPaulBrewer/gridvoting#27]
## Can power method solver be adapted to zero out dominated items?

This would involve a different final step than averaging over all rows of P^power.

Consider this utility profile for 3 voters over 6 items:

```
[
    [1000,900,800,20,10,1],
    [800,1000,900,1,20,10],
    [900,800,1000,10,1,20]
]
```

There are two cycles, but the cycle in the last 3 items is dominated by every item in the cycle in the first 3 items.

Therefore it should have stationary distribution `[1/3,1/3,1/3,0,0,0]`

but instead the power method software shows very small positive probabilities for the last 3 items:

```
vm.stationary_distribution
array([3.33333333e-01, 3.33333333e-01, 3.33333333e-01, 5.67137278e-27,
       5.67137278e-27, 5.67137278e-27])
```

The algebraic solution provides something closer:

```
print(vm.MarkovChain.solve_for_unit_eigenvector())
[ 0.33333333  0.33333333  0.33333333 -0.         -0.         -0.        ]
```

In python minus zero `-0. == 0.` and zero are both equal to zero, but for visual purposes it would be nice to have 0 not -0.

**Comment by DrPaulBrewer** [comment]
Commit: 9dea6e1 has tests for this case for the power method and all permutations.

Passing this test required upgrading the power method solver to examine all rows of the P_power matrix at each step instead of watching two specific rows.

**Comment by DrPaulBrewer** [comment]
closed by PR #28 v0.70
